test/studies/shootout/submitted/regexredux2.chpl

```diff
@@ -30,7 +30,7 @@ proc main(args: [] string) {
   const initLen = data.size;
 
   // remove newlines
-  data = compile(">.*\n|\n").sub("", data);
+  data = data.replace(compile(">.*\n|\n"), "");
```

Comment:
Remember that we want to keep the code in the submitted shootouts directory in line with the CLBG site, so these should have their .good/.perfkeys files updated with the deprecation message instead